src/utils/dates.js

```diff
@@ -145,12 +153,18 @@ let dates = {
     return dates.startOf(new Date(), 'day')
   },
 
-  yesterday() {
-    return dates.add(dates.startOf(new Date(), 'day'), -1, 'day')
+  tomorrow(date) {
+    const tomorrow = new Date(date.valueOf())
+    tomorrow.setDate(tomorrow.getDate() + 1)
+
+    return tomorrow
   },
 
-  tomorrow() {
-    return dates.add(dates.startOf(new Date(), 'day'), 1, 'day')
+  yesterday(date) {
```

Comment:
`dayBefore`...are there any existing usages of this that would have broke?

Reply:
It wasn't used anywhere, the part that used it was probably changed